src/lib/search/searchHelpers.ts

```diff
@@ -137,3 +137,14 @@ export function mapIndexToMessage(index: SearchableIndex) {
       return "";
   }
 }
+
+/**
+ * Meilisearch has constraints on the id field:
+ * it can only contain alphanumeric characters (a-z, A-Z, 0-9), hyphens (-), and underscores (_).
+ * So we need to convert the prisma id to a meilisearch id.
+ * As of now, the Positions table has a prisma id that is not compatible with meilisearch.
+ * It contains dots (.) and possibly swedish characters (åäö).
+ */
+export function prismaIdToMeiliId(id: string) {
+  return btoa(id).replace(/\+/g, "-").replace(/\//g, "_").replace(/=/g, "");
+}
```

Comment:
I changed this function. MD5 seems like it could be [slow](https://stackoverflow.com/questions/6122571/simple-non-secure-hash-function-for-javascript#comment14096359_8831937) and something simple like base64 encoding should be fine.

Reply:
Nice 👍 it doesn't need to be a fancy hashing funciton